src/main/java/org/team1540/robot2023/commands/grabber/GrabberIntake.java

```diff
@@ -0,0 +1,30 @@
+package org.team1540.robot2023.commands.grabber;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.team1540.robot2023.Constants;
+
+public class GrabberIntake extends CommandBase {
```

Comment:
GrabberIntake implies to me that this would be the subsystem. I think using the Command suffix on their names will do a lot to make this clearer